kolibri/core/assets/src/composables/useUserSyncStatus.js

```diff
@@ -0,0 +1,74 @@
+import { ref, onMounted, onUnmounted, computed } from 'kolibri.lib.vueCompositionApi';
+import { UserSyncStatusResource } from 'kolibri.resources';
+import store from 'kolibri.coreVue.vuex.store';
+
+const status = ref(null);
+const queued = ref(false);
+const lastSynced = ref();
+const deviceStatus = ref(null);
+const deviceStatusSentiment = ref(null);
+
+const usageCount = ref(0);
+const timeoutId = ref(null);
+
+export function useUser() {
+  const isUserLoggedIn = computed(() => store.getters.isUserLoggedIn);
+
+  return {
+    isUserLoggedIn,
+  };
+}
+
+export function fetchUserSyncStatus(store, param) {
```

Comment:
It does not appear that store should be an argument for this function. What line 38 sends looks like the value for `param`